kmir/src/tests/integration/test_prove.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import sys
+from pathlib import Path
+from typing import Final
+
+import pytest
+from pytest import LogCaptureFixture
+
+from kmir.__main__ import exec_prove
+
+from ..utils import REPO_ROOT
+
+sys.setrecursionlimit(10**8)
+
+# -------------------
+# Test specifications
+# -------------------
+
+TEST_DIR: Final = REPO_ROOT / 'kmir/src/tests'
+SYMBOLIC_TEST_DIR: Final = TEST_DIR / 'integration/proofs'
+INITIAL_SPECS: Final = SYMBOLIC_TEST_DIR / 'simple-spec.k'
+
+ALL_TESTS: Final = [INITIAL_SPECS]
```

Comment:
At the moment there is only one spec, but I thought I would roughly keep the infrastructure to add more